### PR TITLE
Br improve crops.obic

### PR DESCRIPTION
**Pull request by BrentWHR**
Descriptions of crops.obic need to be added in tables.R #87 
crops.obic needs to be updated with new crop codes for 2020 #72 
Scientific name of crops is required for functions related to pests and diseases.
crops.obic should be saved as csv so changes in the file can be tracked.

### CODE REVIEW

**Comment by BrentWHR** [comment]
@SvenVw What are nf_clay etc.? They lack descriptions and I don't know how to assign values to these columns for the crops introduced in 2020.

**Comment by SvenVw** [comment]
`nf_clay` is used for the nitrogen leaching function. I think that this has been replaced by the  `nf_` values in the `nleach_table`, but that has to be checked

File: R/alter_crops_obic.R [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+cr[grepl('antedeschia', cr$crop_name),crop_name_scientific:= 'zantedeschia aethiopica']
+cr[grepl('maryllis', cr$crop_name),crop_name_scientific:= 'amaryllis belladonna']
+cr[grepl('lauw druifje', cr$crop_name),crop_name_scientific:= 'muscari botryoides']
+cr[grepl('aleriaan', cr$crop_name),crop_name_scientific:= 'valeriana officinalis']
+cr[grepl('Knoflook', cr$crop_name),crop_name_scientific:= 'allium sativum']
+cr[grepl('Quinoa', cr$crop_name),crop_name_scientific:= 'chenopodium quinoa']
+cr[grepl('Pastinaak', cr$crop_name),crop_name_scientific:= 'pastinaca sativa']
+cr[grepl('Pioenroos', cr$crop_name),crop_name_scientific:= 'paeonia']
+cr[grepl('Lavas', cr$crop_name),crop_name_scientific:= 'levisticum officinale']
+cr[grepl('Oregano', cr$crop_name),crop_name_scientific:= 'Origanum vulgare']
```
Comment (SvenVw) [inline R/alter_crops_obic.R:263]: Also lowercase?

**Comment by SvenVw** [comment]
Which scientific name do you give when a brp crop is a collection of multiple crops e.g. `drachtplanten`

**Comment by BrentWHR** [comment]
> 
> 
> Which scientific name do you give when a brp crop is a collection of multiple crops e.g. `drachtplanten`

In such cases where the species or even genus is unknown (as with nature or landscape features) scientific name == 'overig'.
Except when it is clear it is some kind of grass cultivation (without a specific species), then scientific name == 'gras overig'

**Comment by BrentWHR** [comment]
> 
> 
> `nf_clay` is used in the nitrogen leaching function. I think that this has been replaced by the `nf_` values in the `nleach_table`, but that has to be checked

I think nf_clay etc. in crops.obic are still used in calc_nleach():
`  # select the allowed effective N dose (in Dutch: N-gebruiksnorm), being dependent on soil type and region
  sand.south <- c('Zuid-Limburg','Zuidelijk Veehouderijgebied','Zuidwest-Brabant')
  dt[grepl('zand|dal',B_BT_AK), n_eff := nf_sand.other]
  dt[grepl('zand|dal',B_BT_AK) & B_LG_CBS %in% sand.south, n_eff := nf_sand.south]
  dt[grepl('klei',B_BT_AK), n_eff := nf_clay]
  dt[grepl('veen',B_BT_AK), n_eff := nf_peat]
  dt[grepl('loess',B_BT_AK), n_eff := nf_loess]`

**Comment by BrentWHR** [comment]
Started adding crop_season in 9135a20 which is required for #92 .
Still need to decide on category for flower(bulbs) and summer vegetables. 
Maybe some crops should not be assessed such as containerfields, nature...
crop_season will be used to link to another table based on Tabel 2 from Huinink 2018

**Comment by BrentWHR** [comment]
> 
> 
> Still need to decide on category for flower(bulbs) and summer vegetables.

Proposal: flower(bulbs) as "tulp, narcis, hyacint" and summer vegetables as 'witlof, selderij, uien'.
Tulp etc. requires a long season. I don't know how that is for other flowers and bulbs.
I assume summer vegetables require a short season, which is the case for witlof etc.

Seed production fields may also need different seasons than their regular crop. For now, it is assumed seedproduction requires the same season as the main crop.

**Comment by gerardhros** [comment]
code is checked. branch is not merged into master but in release/v1.0.0

**Pull request closed without merging**